views/editor/nav-options.html

```diff
@@ -93,6 +93,13 @@
                 <div class="toggle-button"></div>
               </div>
             </li>
+            <li>
+              <span>{{ gettext("XML/SVG") }}</span>
```

Comment:
where did these strings go? I don't see an update to the locale/en-US/messages.properties file in the PR now, which means `gettext()` won't do anything here.